and-cli.test.js

```diff
@@ -35,21 +36,34 @@ describe("cli", () => {
 
         test("given no commands, it lists each command and description in the commands module", async () => {
             // Arrange & Act
-            const result = await testUtils.executeCliCommand("");
+            const result = await testUtils.executeCliCommand();
 
             // Assert
             commandObjects.forEach((commandObject) => {
                 expect(result).toContain(commandObject.command);
-                expect(result).toContain(commandObject.description);
+
+                // Normally, we'd be able to get away with just comparing the entire description string
+                // against the result output. In the newer versions of Commander, it seems to wrap
+                // longer descriptions onto new lines. We'll just verify that every 'word' of
+                // the description exists in the result string.
+                const descriptionWords = commandObject.description.split(" ");
```

Comment:
Not sure how I feel about this - is there value in ensuring the entire description is displayed, or should we be safe enough just banking on the command string being there? 🤔

Reply:
It's a short test either way, testing for the whole description at least ensures it's not bombing out or getting cut off I guess 🤷‍♂️